core/controllers/base_test.py

```diff
@@ -269,6 +271,31 @@ def test_i18n_keys(self):
                     self.log_line('- %s' % key)
                 self.log_line('')
 
+    def alphabetic_test_i18n_keys(self):
+        """Tests that all JSON files are arranged in aplhabetical order."""
```

Comment:
This should say that it tests that the keys of all the i18n json files are arranged in alphabetical order. Also, "alphabetical" is misspelled.